docs/zh-cn/faq/pipcook-framework.md

```diff
@@ -0,0 +1,17 @@
+# 常见问题
+
+## 我应该在哪里放置我的 pipeline 的配置文件?
+<a id="q1"></a>
```

Comment:
This should be removed.

Reply:
I add this as Anchor point. I think we should not remove it

Reply:
Markdown owns its anchor point for headings, why do we still use this instead?

Reply:
To use heading anchor points, we need to write the whole heading contents after #, which is quite ugly, especially for Chinese version.